perfmetrics/scripts/fetch_metrics.py

```diff
@@ -94,5 +123,15 @@ def _parse_arguments(argv):
     for row in metrics_data:
       vm_metrics_data.append(row)
 
-  if args.upload:
-    gsheet.write_to_google_sheet(VM_WORKSHEET_NAME, vm_metrics_data)
+  # Only some extracted metrics will be uploaded to Google Spreadsheets and BigQuery
+  # Skipping first column as it's a duplicate of 2nd column. Appending 8 none values as those VM metrics are not being extracted currently.
+  vm_metrics_data_upload = [row[1:] + [None]*8 for row in vm_metrics_data]
```

Comment:
Similar to https://github.com/GoogleCloudPlatform/gcsfuse/pull/1193#discussion_r1250704247, move this filtering in fetch_metrics